index.html

```diff
@@ -418,6 +418,42 @@ <h3 class="pivx-bold-title" style="font-size:38px;"><span>Create a new</span>Van
               <!-- // GENERATE VANITY WALLET -->
               <br>
 
+              <!-- IMPORT LEDGER HARDWARE WALLET -->
+              <div id='generateHardwareWallet' class="dashboard-item">
+                <div class="container">
+
+                  <div class="coinstat-icon">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50">
+                      <path d="M45.936 22.101c-1.321-4.06-2.976-6.808-3.045-6.922a.754.754 0 00-1.39.506c.003.016.193 1.655-1.762 3.08-.671.49-1.896.378-2.569-.234-.828-.753-.744-2.081.237-3.738C40.95 8.818 35.582 1.034 35.352.707a.757.757 0 00-1.369.35c-.002.022-.264 2.274-1.601 4.499-1.747 2.906-4.447 4.417-8.026 4.491-5.056.105-8.143 1.798-9.843 3.201-1.885 1.556-2.542 3.111-2.569 3.176a.755.755 0 00.525 1.022c1.916.45 2.546 1.966 2.503 3.127-.038 1.027-.636 2.314-2.201 2.563-1.112.176-1.965-.002-2.535-.532-1.144-1.063-.975-3.236-.973-3.256a.755.755 0 00-1.232-.65c-4.492 3.706-6.797 10.581-5.735 17.109.468 2.875 1.59 5.545 3.247 7.724 1.854 2.437 4.333 4.266 7.372 5.434a.754.754 0 10.542-1.408C6.292 44.8 4.323 38.86 3.787 35.563a18.666 18.666 0 01.711-8.828c.754-2.28 1.901-4.258 3.354-5.803.154.907.518 1.99 1.352 2.77.928.866 2.208 1.176 3.805.922 2.001-.317 3.396-1.923 3.472-3.997.06-1.631-.775-3.485-2.732-4.342.97-1.445 3.855-4.591 10.639-4.732 5.173-.107 7.922-2.926 9.317-5.272.666-1.12 1.09-2.23 1.356-3.125.397.746.855 1.708 1.234 2.79 1.136 3.235 1.072 5.953-.187 8.077-1.94 3.273-.615 5.022.044 5.622 1.212 1.102 3.219 1.252 4.475.337.998-.728 1.583-1.517 1.924-2.229a37.385 37.385 0 011.962 4.852c1.156 3.565 2.237 8.886.819 13.997-1.352 4.873-4.731 8.762-10.044 11.56a.755.755 0 00.704 1.335c5.698-3.001 9.331-7.206 10.796-12.5 1.519-5.484.375-11.128-.852-14.896z"></path>
+                      <path d="M40.953 11.943a.754.754 0 101.459-.376l-.294-1.143a.756.756 0 00-1.462.377l.297 1.142zM25.17 33.984a.848.848 0 000-1.694h-6.537a.117.117 0 00-.117.118v.729c0 .454.358.825.805.853l5.849-.006z"></path>
+                      <path d="M26.779 27.343h-6.921a.117.117 0 00-.118.117v.73a.86.86 0 00.805.853l5.883-.015c.219 0 .419.007.419.007a4.106 4.106 0 013.896 4.103 4.083 4.083 0 01-3.957 4.08h-6.192c-.041 0-.083 0-.124.009a.838.838 0 00-.715.775v8.363c0 .066.055.115.117.115h.729a.859.859 0 00.854-.807v-6.651c0-.057.055-.105.109-.105l5.346-.008a6.014 6.014 0 001.334-.213 5.788 5.788 0 004.205-5.567 5.81 5.81 0 00-5.67-5.786z"></path>
+                    </svg>
+                  </div>
+
+                  <div class="col-md-12 dashboard-title">
+                    <h3 class="pivx-bold-title" style="font-size:38px;"><span>Import your</span>Hardware Wallet</h3>
```

Comment:
nit: "Access" rather than "Import" makes more sense here, just making sure it doesn't seem like MPW is extracting private keys directly from Ledger (we know it won't, and there's a notice below, but from a user perspective!)